AnimatedTextInput/Classes/AnimatedTextInputFieldConfigurator.swift

```diff
@@ -4,7 +4,9 @@ public struct AnimatedTextInputFieldConfigurator {
 
     public enum AnimatedTextInputType {
         case standard
-        case password
+        case passwordWithDisclosure
+        case passwordWithoutDisclosure
```

Comment:
I like to have the `passwordWithoutDisclosure` case, but leave `password` as it was so we do not modify existing api